src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -81,23 +83,35 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                 this.WriteLine($"{"LogRecord.Body:",-RightPaddingLength}{logRecord.Body}");
             }
 
-            if (logRecord.Attributes != null)
+            if (logRecord.Attributes != null || logRecord.Exception != null)
```

Comment:
No need to add this check.

Reply:
The idea here is to treat exception attributes like any other attributes, since [SemConv defines them as attributes](https://opentelemetry.io/docs/specs/semconv/exceptions/exceptions-logs/).

With this, the output is something like this:
```
LogRecord.Attributes (Key:Value):
    OriginalFormat (a.k.a Body): You divided by 0
    exception.type: DivideByZeroException
    exception.message: Attempted to divide by zero.
    exception.stacktrace: System.DivideByZeroException: Attempted to divide by zero.
   at OpenTelemetry.Exporter.Console.Tests.ConsoleLogRecordExporterTest.VerifyExceptionAttributesAreWritten() in /Users/gregkalapos/repos/gregkalapos_opentelemetry-dotnet/test/OpenTelemetry.Exporter.Console.Tests/ConsoleLogRecordExporterTest.cs:line 39
LogRecord.Exception:               System.DivideByZeroException: Attempted to divide by zero.
   at OpenTelemetry.Exporter.Console.Tests.ConsoleLogRecordExporterTest.VerifyExceptionAttributesAreWritten() in /Users/gregkalapos/repos/gregkalapos_opentelemetry-dotnet/test/OpenTelemetry.Exporter.Console.Tests/ConsoleLogRecordExporterTest.cs:line 3
```

If we move the logging of exception attributes outside this block, then the `LogRecord.Attributes (Key:Value):` may not be printed (if no other attribute is present, and I'm not sure we can assume we'll always have any other attribute). Also the indention of these attribute also matches other attribute's indention on purpose.

Reply:
I'm concerned about the readability of the exception callstack, create a tracking issue with my suggestions #5372.

Reply:
I agree with the concerns raised by @reyang here. YAML has better support for multi-line strings. @gregkalapos Would you be interested in working on updating the `ConsoleExporte`r to print the data in `yaml` format?

Reply:
Yeah, I'd be happy to look into that - I also think it's a good idea. Let's handle it in the issue.

What do we do with this PR?

The issue with the readability of the exception callstack is not new, `LogRecord.Exception` is already printed a few lines below with this output (that's not changed in the PR):

```
LogRecord.Exception:               System.DivideByZeroException: Attempted to divide by zero.
   at OpenTelemetry.Exporter.Console.Tests.ConsoleLogRecordExporterTest.VerifyExceptionAttributesAreWritten() in /Users/gregkalapos/repos/gregkalapos_opentelemetry-dotnet/test/OpenTelemetry.Exporter.Console.Tests/ConsoleLogRecordExporterTest.cs:line 39
```

I see https://github.com/open-telemetry/opentelemetry-dotnet/issues/5372 and this PR orthogonal.

Reply:
> The issue with the readability of the exception callstack is not new,
> I see https://github.com/open-telemetry/opentelemetry-dotnet/issues/5372 and this PR orthogonal.

I don't see them as orthogonal. The issue clearly talks about new things getting added with no well-defined structure. Just because the ConsoleExporter already has a readability issue, doesn't mean we keep making it worse.

> What do we do with this PR?

I would say we should close it and invest in making ConsoleExporter better and extensible for newer changes in a well-defined way. I don't think anyone is blocked on ConsoleExporter not printing out the exception information with Semantic Convention attributes. As you previously mentioned, the exception information is already printed to console in some other format.